rootfs/standard/usr/share/mynode_apps/datum/scripts/install_datum.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+source /usr/share/mynode/mynode_device_info.sh
+source /usr/share/mynode/mynode_app_versions.sh
+
+set -x
+set -e
+
+echo \"==================== INSTALLING APP ====================\"
+
+# export bitcoin password
+BTCPSW=$(cat /mnt/hdd/mynode/settings/.btcrpcpw)
+
+# git clone datum repo
+git clone https://github.com/OCEAN-xyz/datum_gateway.git .
+
+# verify datum
+curl -L "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x1a3e761f19d2cc7785c5502ea291a2c45d0c504a" | gpg --import
+git verify-tag v0.3.0beta
```

Comment:
Last thing, you should be able to replace "v0.3.0" in this script with ${VERSION} so the script will always pull the version set in the json file. That way just bumping that version will update the app.